apps/codeforafrica/src/components/NavListItem/index.js

```diff
@@ -0,0 +1,96 @@
+import { Link } from "@commons-ui/next";
+import { styled } from "@mui/material/styles";
+import PropTypes from "prop-types";
+import * as React from "react";
+
+const NavLinkItemRoot = styled("li")(
+  ({ theme: { breakpoints, typography } }) => ({
+    margin: `${typography.pxToRem(20)} 0`,
+    listStyle: "none",
+    [breakpoints.up("lg")]: {
+      margin: `${typography.pxToRem(10)} 0`,
+      listStyle: "none",
+    },
+  })
+);
+
+const NavLink = styled(Link, {
+  slot: "Root",
+  overridesResolver: (props, styles) => {
+    const { ownerState } = props;
+    return [styles.root, ownerState.direction && styles.direction];
+  },
+})(({ theme: { breakpoints, palette, typography }, ownerState }) => ({
+  textTransform: "capitalize",
+  color: "white",
+  textDecoration: "none",
+  fontSize: typography.pxToRem(28),
+  backgroundColor: "transparent",
+  "&:hover, &:active, &:focus, &:focus-within": {
+    color: "white",
+  },
+  [breakpoints.up("lg")]: {
+    textTransform: "capitalize",
+    color: "white",
+    textDecoration: "none",
+    fontSize: typography.pxToRem(23),
+    margin: typography.pxToRem(24),
+    fontweight: 700,
+    lineHeight: typography.pxToRem(28),
+    backgroundColor: "transparent",
+    borderColor: "none",
+    "&:hover, &:active, &:focus, &:focus-within": {
+      color: "white",
+    },
+  },
```

Comment:
Rather than doing it this way, either:

1. Use known typography _variants_ props the component since `Link` supports `variant`, `color`, etc., or
2. Use `sx` prop since it allows us to switch typography, color, etc., based on screen size as we..